integration-tests/src/test/resources/queries/wikipedia_editstream_queries.json

```diff
@@ -373,7 +373,7 @@
             "context": {
                 "useCache": "true",
                 "populateCache": "true",
-                "timeout": 60000
+                "timeout": 180000
```

Comment:
with shorter timeouts, this part of the test was failing.  I asked about this a while back at the dev meeting and the answer seemed to be that we're not trying to measure performance with these tests and to not be concerned about it unless it was outrageous.  I made the timeout much bigger so that we'd quit seeing what would be considered spurious failures.  If we think there might be a problem with this topN query, perhaps we could investigate it under a separate issue.